src/main/java/edu/kis/powp/jobs2d/drivers/adapter/HeadMoverAdapter.java

```diff
@@ -1,18 +1,23 @@
 package edu.kis.powp.jobs2d.drivers.adapter;
 
+import edu.kis.legacy.drawer.panel.DefaultDrawerFrame;
 import edu.kis.legacy.drawer.panel.DrawPanelController;
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class HeadMoverAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect, the purpose of the adapter or what was actually adapted.
Inheritance is not used (no super drawLine calls). Should be removed!